pyabc/distance/scales.py

```diff
@@ -19,113 +19,134 @@
     """
-    data = np.asarray(kwargs['data'])
     std = np.std(data)
     return std
 
 
-def bias(**kwargs):
+def bias(data, x_0, **kwargs):
     """
     Bias of sample to observed value.
     """
-    data = np.asarray(kwargs['data'])
-    x_0 = kwargs['x_0']
     bias = np.abs(np.mean(data) - x_0)
     return bias
 
 
-def root_mean_square_deviation(**kwargs):
+def root_mean_square_deviation(data, x_0, **kwargs):
     """
     Square root of the mean squared error, i.e.
     of the bias squared plus the variance.
     """
-    bs = bias(**kwargs)
-    std = standard_deviation(**kwargs)
+    bs = bias(data, x_0)
+    std = standard_deviation(data)
     mse = bs**2 + std**2
     rmse = np.sqrt(mse)
     return rmse
 
 
-def median_absolute_deviation_to_observation(**kwargs):
+def median_absolute_deviation_to_observation(data, x_0, **kwargs):
     """
     Median absolute deviation of data w.r.t. the observation x_0.
     """
-    data = np.asarray(kwargs['data'])
-    x_0 = kwargs['x_0']
+    data = np.array(data)
     mado = np.median(np.abs(data - x_0))
     return mado
 
 
-def mean_absolute_deviation_to_observation(**kwargs):
+def mean_absolute_deviation_to_observation(data, x_0, **kwargs):
     """
     Mean absolute deviation of data w.r.t. the observation x_0.
     """
-    data = np.asarray(kwargs['data'])
-    x_0 = kwargs['x_0']
+    data = np.array(data)
     mado = np.mean(np.abs(data - x_0))
     return mado
 
 
-def combined_median_absolute_deviation(**kwargs):
+def combined_median_absolute_deviation(data, x_0, **kwargs):
     """
     Compute the sum of the median absolute deviations to the
     median of the samples and to the observed value.
     """
-    mad = median_absolute_deviation(**kwargs)
-    mado = median_absolute_deviation_to_observation(**kwargs)
+    mad = median_absolute_deviation(data)
+    mado = median_absolute_deviation_to_observation(data, x_0)
     cmad = mad + mado
     return cmad
 
 
-def combined_mean_absolute_deviation(**kwargs):
+def combined_mean_absolute_deviation(data, x_0, **kwargs):
     """
     Compute the sum of the mean absolute deviations to the
     mean of the samples and to the observed value.
     """
-    mad = mean_absolute_deviation(**kwargs)
-    mado = mean_absolute_deviation_to_observation(**kwargs)
+    mad = mean_absolute_deviation(data)
+    mado = mean_absolute_deviation_to_observation(data, x_0)
     cmad = mad + mado
     return cmad
 
 
-def standard_deviation_to_observation(**kwargs):
+def standard_deviation_to_observation(data, x_0, **kwargs):
     """
     Standard deviation of absolute deviations of the data w.r.t.
     the observation x_0.
     """
-    data = np.asarray(kwargs['data'])
-    x_0 = kwargs['x_0']
+    data = np.array(data)
     stdo = np.std(np.abs(data - x_0))
     return stdo
+
+
+def span(data, **kwargs):
+    """
+    Compute the difference of largest and smallest data point.
+    """
+    return max(data) - min(data)
+
+
+def mean(data, **kwargs):
```

Comment:
Why do you define this function if in the previous instances you use `np.mean` instead of this new defined `mean` function?